docs/semver.md

```diff
@@ -21,6 +21,9 @@ versions:
 - Support for any of Shadow's [supported platforms](supported_platforms.md)
   will not be dropped, unless those platforms no longer receive free updates
   and support from the distribution's developer.
+- We will not increase the minimum Linux kernel version beyond the version of the
+General Availability kernel for all supported platforms [supported
+platforms](supported_platforms.md).
```

Comment:
"supported platforms" is stated twice. Also maybe add a disclaimer ("some exceptions apply")?

Reply:
Or maybe don't mention the "General Availability kernel" and just say that we won't increase it beyond what is stated in the supported platforms doc.

Reply:
Tweaked the wording here - wdyt?

Reply:
Looks good to me.